x-pack/plugins/lists/common/schemas/types/comment.ts

```diff
@@ -3,26 +3,33 @@
  * or more contributor license agreements. Licensed under the Elastic License;
  * you may not use this file except in compliance with the Elastic License.
  */
+
+/* eslint-disable @typescript-eslint/camelcase */
+
 import * as t from 'io-ts';
 
-export const comments = t.intersection([
+import { NonEmptyString } from '../../siem_common_deps';
+import { created_at, created_by, id, updated_at, updated_by } from '../common/schemas';
+
+export const comment = t.intersection([
   t.exact(
     t.type({
-      comment: t.string,
-      created_at: t.string, // TODO: Make this into an ISO Date string check,
-      created_by: t.string,
+      comment: NonEmptyString,
+      created_at,
```

Comment:
Moved to use the same ones that are in common, so when we update those to be ISO Date types, these are good to go as well.